Core/ConfigValues.h

```diff
@@ -31,6 +31,22 @@ const int PSP_DEFAULT_FIRMWARE = 660;
 static const int8_t VOLUME_OFF = 0;
 static const int8_t VOLUME_FULL = 10;
 
+struct ConfigTouchPos {
```

Comment:
Originally, the intent of this was to keep these separate from Config.h so not everything had to include both (it's so expensive to change Config.h too, everything needs recompile.)  But now that this is included by Config.h anyway, maybe we should just merge it into Config.h...

-[Unknown]

Reply:
Moving these two structs doesn't really change that, it's still possible to include just the values file (actually even more). But yeah, eventually would be nice to use real enums instead of ints for the values in Config if we can find a way to do that, and in that case we'd always need to include both anyway.